doc/index.md

```diff
@@ -1,25 +1,213 @@
+# Panel: The powerful data app solution for Python
```

Comment:
Thinking about it. We could also use the header "More than a powerful data app solution for Python". Because Panel is not only for data apps. But its also a collection of carefully selected components and carefully crafted apis for working with data exploration and data visualization in notebooks and IDEs.